filebeat/autodiscover/builder/hints/logs.go

```diff
@@ -60,7 +61,7 @@ func NewLogHints(cfg *common.Config) (autodiscover.Builder, error) {
 		return nil, fmt.Errorf("unable to unpack hints config due to error: %v", err)
 	}
 
-	moduleRegistry, err := fileset.NewModuleRegistry([]*common.Config{}, "", false)
+	moduleRegistry, err := fileset.NewModuleRegistry([]*common.Config{}, beat.Info{}, false)
```

Comment:
```suggestion
	moduleRegistry, err := fileset.NewModuleRegistry(nil, beat.Info{}, false)
```

Might as well fix that while were making a change.